napari/napari
# Moved files to agree with ToC reorganization.

This PR should be merged before napari/napari.github.io#299

# Description
Creates new docs structure to agree with the new ToC organization. This also touches `docs/copy_docs.py::update_toc`, to make sure it doesn't skip toc items.

## Type of change


# References
Addresses #3390

# How has this been tested?
To test locally, you can:
1. Clone both [napari/napari](https://github.com/napari/napari) and [napari/napari.github.io](https://github.com/napari/napari.github.io)
2. Go into the napari/napari folder and run
   ```bash
   $ pip install -e '.[all]'
   $ make docs
   ```
3. View changes rendered in "_build/html"
4. Run `python docs/copy-docs.py` (this will copy the appropriate sources into the napari/napari.github.io folder and update the `napari/napari.github.io/_toc.yml` file
5. Go into the napari/napari.github.io folder and run
	```bash
	$ yarn dev
	```
6. Confirm that this command renders the appropriate outputs.

Note that napari/napari.github.io#299 needs to also be merged to render the full documentation reorganization.

## Final checklist:
- [x] My PR is the minimum possible work for the desired functionality
- [ ] If I included new strings, I have used `trans.` to make them localizable.
      For more information see our [translations guide](https://napari.org/docs/dev/guides/translations.html).
